docs/windows.md

```diff
@@ -254,11 +254,17 @@ Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
 Firstly, install wins, kubelet, kubeadm using script `PrepareNode.ps1` provided
 by kubernetes. The third component [`wins`](https://github.com/rancher/wins) is
 used to run kube-proxy and antrea-agent on Windows host inside the Windows
-container. The following command downloads and executes `PrepareNode.ps1`:
+container. Specify the Node IP, Kubernetes Version and Container Runtime while
+running the script. If you do not specify any container runtime, it will be Docker
```

Comment:
I would prefer containerd to work by default since docker is deprecated since K8s 1.24